inc/FSMCIO.h

```diff
@@ -1,32 +1,17 @@
 #ifndef FSMCIO_H
 #define FSMCIO_H
 
-#include "ST7735.h"
-#include "ZPin.h"
-
-#ifdef FSMC_Bank1
+#include "SPI.h"
+#include "Pin.h"
+#include "ScreenIO.h"
 
 namespace codal
 {
 
-class FSMCIO : public ScreenIO
-{
```

Comment:
I don't think you should remove the class definition, there might be something added in the future, right?

Why not just have a static instance member variable on the ScreenIO interface?

Reply:
When we need to have additional members in the future, we can move it back I guess. I thought you guys always liked hiding implementation details ;)

I had a HAL descriptor in there that wasn't present when compiling for 401 but I guess this is gone now.